spec/datetimeformat.html

```diff
@@ -379,7 +400,18 @@
       <emu-alg>
         1. If NewTarget is *undefined*, let _newTarget_ be the active function object, else let _newTarget_ be NewTarget.
         1. Let _dateTimeFormat_ be ? OrdinaryCreateFromConstructor(_newTarget_, `"%DateTimeFormatPrototype%"`, &laquo; [[initializedIntlObject]], [[initializedDateTimeFormat]], [[locale]], [[calendar]], [[numberingSystem]], [[timeZone]], [[weekday]], [[era]], [[year]], [[month]], [[day]], [[hour]], [[minute]], [[second]], [[timeZoneName]], [[hour12]], [[hourNo0]], [[pattern]], [[boundFormat]] &raquo;).
-        1. Return ? InitializeDateTimeFormat(_dateTimeFormat_, _locales_, _options_).
+        1. Perform ? InitializeDateTimeFormat(_dateTimeFormat_, _locales_, _options_).
+      </emu-alg>
+      <emu-normative-optional><span class="normative-optional">Normative Optional</span><div class="normative-optional-contents">
+      <emu-alg>
+        4. Let _this_ be the *this* value.
+        1. If NewTarget is *undefined* and ? InstanceofOperator(_this_, %NumberFormat%),
+          1. Perform ? DefineOwnPropertyOrThrow(_this_, Intl.[[FallbackSymbol]], { [[Value]]: _dateTimeFormat_, [[Writable]]: *false*, [[Enumerable]]: *false*, [[Configurable]]: *false* }).
```

Comment:
@anba will something like this be sufficient? 

![image](https://cloud.githubusercontent.com/assets/38969/22834232/7a6b3990-ef82-11e6-9a3a-494cdb072f64.png)

or should we just throw a TypeError if this-value is not an object before calling DefineOwnPropertyOrThrow?